src/clarity-angular/datagrid/datagrid-column.ts

```diff
@@ -43,10 +44,16 @@ export class DatagridColumn extends DatagridFilterRegistrar<DatagridStringFilter
         super(filters);
         this._sortSubscription = _sort.change.subscribe(sort => {
             // We're only listening to make sure we emit an event when the column goes from sorted to unsorted
```

Comment:
Is this comment still accurate? It doesn't seem like that is or should be the case. Shouldn't `sortOrderChange` emit when I reverse the sort order too?